osu.Game/Screens/Edit/EditorClock.cs

```diff
@@ -239,20 +263,10 @@ private void updateSeekingState()
                 // we are either running a seek tween or doing an immediate seek.
                 // in the case of an immediate seek the seeking bool will be set to false after one update.
                 // this allows for silencing hit sounds and the likes.
-                seekingOrStopped.Value = Transforms.Any();
+                seekingOrStopped.Value = IsSeeking;
```

Comment:
Thoughts about splitting `seekingOrStopped` into two `seeking` and `stopped` bindables, and then substituting what is the current `seekingOrStopped` with an `AggregateBindable`? I guess this is not *horrible* as it is right now but I still find this song and dance about having to sorta track both simultaneously a little teeth-pain-inducing.

Reply:
alternative thought would be to have a tri-state enum (seeking, stopped and playing seem like mutually exclusive states)